### PR TITLE
docs: Collect release notes for F41

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks! :)

File: docs/release-notes.rst [hunk 1]
```diff
@@ -5,6 +5,110 @@ This document describes major installer-related changes in Fedora releases.
...
+Remote repository for Flatpaks after deployment are now configurable
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Currently when OSTree installation detects Flatpak repository in the installation media
+these Flatpaks are deployed and the remote was hardcoded to remote Fedora. This remote
+is then used for updating the Flatpaks after installation.
+
+After this change Flatpak remote can be set by ``flatpak_remote`` key in the configuration
+file.
+
```
Comment (jkonecny12) [inline docs/release-notes.rst:44]:
```suggestion

See also:
    - https://github.com/rhinstaller/anaconda/pull/5493
    
```

File: docs/release-notes.rst [hunk 2]
```diff
@@ -5,6 +5,110 @@ This document describes major installer-related changes in Fedora releases.
...
+available in old RHELs and Fedoras.
+
+See also:
+    - https://bugzilla.redhat.com/2276036
+    - https://github.com/rhinstaller/anaconda/pull/5628
+
+Do not create default network profiles for network port devices
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Traditionally Anaconda creates default persistent network profiles (ifcfg files or keyfiles) for every supported wired network device.  We would like to move towards creating profiles only for devices explicitly configured by installer. As a step in this direction do not create such files for devices used as ports of a virtual device (for example bond device) configured by installer, unless they were explicitly configured separately (for example in early stage from boot options).
```
Comment (jkonecny12) [inline docs/release-notes.rst:83]:
```suggestion
Traditionally Anaconda creates default persistent network profiles (ifcfg files or keyfiles) for every supported wired network device. We would like to move towards creating profiles only for devices explicitly configured by installer. As a step in this direction do not create such files for devices used as ports of a virtual device (for example bond device) configured by installer, unless they were explicitly configured separately (for example in early stage from boot options).
```

**Review by jkonecny12 — CHANGES_REQUESTED** [review]
A few notes from my side.

Also I think we should make the lines consistent. Currently every part has different line size. It seems more or less consistent in the rest of the document.

**Comment by KKoukiou** [comment]
> A few notes from my side.
> 
> Also I think we should make the lines consistent. Currently every part has different line size. It seems more or less consistent in the rest of the document.

Done. Good point.

**Comment by jkonecny12** [comment]
Seems there is a commit which shouldn't be here. Otherwise it looks good.

**Comment by KKoukiou** [comment]
> Seems there is a commit which shouldn't be here. Otherwise it looks good.

I was just not rebased.

**Review by jkonecny12 — APPROVED** [review]
LGTM!

**Pull request merged**